network/nxos/nxos_igmp.py

```diff
@@ -61,21 +61,28 @@
         choices: ['present', 'default']
 '''
 EXAMPLES = '''
-# default igmp global params (all params except restart)
-- nxos_igmp: state=default host={{ inventory_hostname }}
-# ensure the following igmp global config exists on the device
-- nxos_igmp: flush_routes=true enforce_rtr_alert=true host={{ inventory_hostname }}
-# restart the igmp process
-- nxos_igmp: restart=true host={{ inventory_hostname }}
-'''
-
-EXAMPLES = '''
-# configure a simple asn
-- nxos_bgp:
-      asn=65535
-      vrf=test
-      router_id=1.1.1.1
-      state=present
+- name: Default igmp global params (all params except restart)
+  nxos_igmp:
+    state: default
+    host: "{{ inventory_hostname }}"
+
+- name: Ensure the following igmp global config exists on the device
+  nxos_igmp:
+    flush_routes: true
+    enforce_rtr_alert: true
+    host: "{{ inventory_hostname }}"
+
+- name: Restart the igmp process
+  nxos_igmp:
+    restart: true
+    host: "{{ inventory_hostname }}"
+
+- name: Configure a simple ASN
+  nxos_bgp:
```

Comment:
I removed this example. Seems to be a copy/paste error from the `nxos_bgp` module.